Shared/Realm.Shared/Linq/IRealmCollection.cs

```diff
@@ -113,6 +113,7 @@ public interface IRealmCollection<out T> : IReadOnlyList<T>, INotifyCollectionCh
         /// Gets the <see cref="ObjectSchema"/> of the contained objects.
         /// </summary>
         /// <value>The ObjectSchema of the contained objects.</value>
+        [Obsolete("This property is no longer necessary and will be removed in a subsequent release.")]
```

Comment:
Meh, I'd be surprised if anyone ever used it. I'll revert it though as it's not directly related to this PR (I'll still obsolete it in another one 👻)